org.eclipse.winery.frontends/app/tosca-management/src/app/wineryTableModule/wineryTable.component.ts

```diff
@@ -119,6 +119,8 @@ export class WineryTableComponent implements OnInit, DoCheck {
 
     @ViewChild('tableContainer') tableContainer: any;
     @Input() title: string;
+    @Input() titleRouterLink?: any[] | string;
```

Comment:
Why is this an array of `any`?

Reply:
the definition is acutally taken from `RouterLinkWithHref`

```
export declare class RouterLinkWithHref implements OnChanges, OnDestroy {
    private router;
    private route;
    private locationStrategy;
    target: string;
    queryParams: {
        [k: string]: any;
    };
    fragment: string;
    queryParamsHandling: QueryParamsHandling;
    preserveFragment: boolean;
    skipLocationChange: boolean;
    replaceUrl: boolean;
    state?: {
        [k: string]: any;
    };
    private commands;
    private subscription;
    private preserve;
    href: string;
    constructor(router: Router, route: ActivatedRoute, locationStrategy: LocationStrategy);
    routerLink: any[] | string;
    preserveQueryParams: boolean;
    ngOnChanges(changes: {}): any;
    ngOnDestroy(): any;
    onClick(button: number, ctrlKey: boolean, metaKey: boolean, shiftKey: boolean): boolean;
    private updateTargetUrlAndHref;
    readonly urlTree: UrlTree;
}

```

Here is an example from the docs
> For instance ['/team', teamId, 'user', userName, {details: true}] means that we want to generate a link to /team/11/user/bob;details=true.


See `org.eclipse.winery.frontends/node_modules/@angular/router/src/directives/router_link.d.ts`